app/src/androidTest/java/ru/startandroid/organizer/WeatherDataInstrumentedTest.kt

```diff
@@ -16,7 +16,7 @@ import org.junit.Assert.*
  * @see [Testing documentation](http://d.android.com/tools/testing)
  */
 @RunWith(AndroidJUnit4::class)
-class ExampleInstrumentedTest {
+class WeatherDataInstrumentedTest {
```

Comment:
Why do we need this renaming? (and a few other similar renaming).

Reply:
I used a website to create a POJO for Json parsing. The default name for the main class was "Example". So I decided to rename it into a form with some meaning (Examle -> WeatherData). So, as we can see, the "test" files were renamed automatically.

Reply:
OK, let's remove these tests file.